common/src/test/java/com/revenuecat/purchases/common/EntitlementInfosTests.kt

```diff
@@ -1,7 +1,10 @@
-package com.revenuecat.purchases
+package com.revenuecat.purchases.common
```

Comment:
Tests are only able to import public stuff from other modules, or public/internal from their same module. In this case, this test is using `buildPurchaserInfo` which lives in `common` and that's why this needs to be either in `common` or in `purchases`. Typically tests would live in the same module as the class they are testing. In this case, this test should actually live in the `public` module, where `EntitlementInfos` is and use an internal constructor of `PurchaserInfo` instead of `buildPurchaserInfo`. That way if you ever want to trash this class, everything is isolated in the same module, and nothing leaks outside.